README.md

```diff
@@ -24,13 +24,24 @@ For speed, it's recommended that your current environment has mamba installed
 
 ```
 conda config --set channel_priority flexible  # Make sure your conda config uses flexible channel packaging to prevent unsatisfiable errors
-conda create -n mamba -c conda-forge mamba
+conda create -n mamba -c conda-forge mamba python=3.11
 conda activate mamba
-mamba install -c conda-forge -c bioconda snakemake
+mamba install -c conda-forge -c bioconda snakemake=7
 ```
 
 ## Apply model
-Modify `config/config_biosamples.tsv` with your multiome data, including RNA_matrix, ATAC_matrix, and candidate_pairs files. 
+Before running this workflow, users should perform clustering to define cell clusters through regular single-cell analysis (such as Seurat & Signac).
+Required input data includes (refer to the example data in the `example_chr22_multiome` folder):
+1. Combined meta_data, including columns for cell_name, sample, cluster, and barcode.
+2. Combined RNA matrix, normalized by the total UMI count of each cell. It can be with or without log transformation.
+3. Indexed fragment files for each sample.
+
+To configure the pipeline:
+- Modify `config/config.yaml` to specify paths for meta_data, rna_matrix, and results_dir.
```

Comment:
As far as I understand, we're only equipped to run the repo against 1 experiment, which may have multiple clusters. Thinking ahead, we'd probably want to run this @ scale, which means being able to support running multiple experiments. This PR can focus on running on 1 experiment, but we should support multiple experiments in a later PR